test/units/parsing/test_mod_args.py

```diff
@@ -137,4 +134,4 @@ def test_bogus_action(self):
         with pytest.raises(AnsibleParserError) as err:
             m.parse()
 
-        assert err.value.args[0].startswith("couldn't resolve module/action 'bogusaction'")
+        assert err.value.args[0].startswith(f"couldn't resolve module/action \'{', '.join(args_dict.keys())}\'")
```

Comment:
This error message only includes the first dictionary key:

```suggestion
        assert err.value.args[0].startswith(f"couldn't resolve module/action '{next(iter(args_dict))}'")
```